cmake/CMakeLists.txt

```diff
@@ -348,6 +348,13 @@ if(PKG_LATTE)
   list(APPEND LAMMPS_LINK_LIBS ${LATTE_LIBRARIES} ${LAPACK_LIBRARIES})
 endif()
 
+if(PKG_USER-SCAFACOS)
+    FIND_PACKAGE(PkgConfig)
+    PKG_CHECK_MODULES(SCAFACOS scafacos)
```

Comment:
Need to add `REQUIRED` here.

And we need to add a `DOWNLOAD_SCAFACOS` option.